can/io/blf.py

```diff
@@ -264,8 +265,8 @@ def _parse_data(self, data):
                 continue
 
             # Calculate absolute timestamp in seconds
-            factor = 1e-5 if flags == 1 else 1e-9
-            timestamp = timestamp * factor + start_timestamp
+            factor = Decimal("1e-5") if flags == 1 else Decimal("1e-9")
+            timestamp = float(Decimal(timestamp) * factor) + start_timestamp
```

Comment:
I think that can be achieved without Decimal:
```python3
            factor = 100_000 if flags == 1 else 1_000_000_000
            timestamp = (timestamp + round(start_timestamp * factor)) / factor
```

Reply:
I agree that we can achieve it without `Decimal`.
But what is the downside of using `Decimal`?

IMO, using the Python built-in module `Decimal` makes it very obvious what we are doing here.